playbooks/main.yml

```diff
@@ -61,3 +61,6 @@
 
 - import_playbook: ocp-iscsi.yml
   when: ocp_iSCSI_enabled
+
+- import_playbook: monitoring-user-defined-projects.yml
+  when: enable_user_defined_project_monitoring
```

Comment:
Can you check if variable is defined and if it is true? This way you don't need to make it a required variable and also allow backward compatibility